sdk/nodejs/tests/provider/experimental/testdata/array-types/index.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2025-2025, Pulumi Corporation.  All rights reserved.
+
+import * as pulumi from "@pulumi/pulumi";
+
+export interface MyComponentArgs {
+    anArrayOfStrings: Array<string>;
+    anArrayOfNumbers?: Array<number>;
+    anArrayOfBooleans: Array<boolean>;
+    inputArrayOfStrings: pulumi.Input<Array<string>>;
+    inputArrayOfNumbers?: pulumi.Input<Array<number>>;
+    inputArrayOfBooleans: pulumi.Input<Array<boolean>>;
```

Comment:
Shouldn't the three above produce a plain `items` type?

Reply:
Hrm I thought not, but looking at the python code they indeed are there and probably should. Will change.